index.js

```diff
@@ -73,9 +32,10 @@ page.onError = function(msg, trace) {
 
 function scrapPage(status) {
     console.log("****Status: " + status + "****\n");
+    console.log(fotocasaSpider);
```

Comment:
I think we should remove this `console.log`

Reply:
woops haha se coló

Reply:
Done